web/src/components/alerting.tsx

```diff
@@ -1118,6 +1118,7 @@ const PollerPages = () => {
         <Route path="/dev-monitoring/ns/:ns/alerts" exact component={AlertsPage} />
         <Route path="/dev-monitoring/ns/:ns/alerts/:ruleID" component={AlertsDetailsPage} />
         <Route path="/dev-monitoring/ns/:ns/metrics" exact component={QueryBrowserPage} />
+        <Route path="/dev-monitoring/ns/:ns/" component={MonitoringDashboardsPage} />
```

Comment:
I don't believe that the dashboards page (or the metrics page above it) need to be in the `PollerPages` component. They don't need to fetch the alerts and rules info, so they can be moved back down to the `MonitoringUI` component.

Reply:
moved both Metrics and Dashboard Routes into <MonitoringUI /> in the latest commit